api/internal/handler/router/routes.go

```diff
@@ -1,17 +1,19 @@
-package routing
+package router
 
 import (
-	"backend/api/handler"
+	handler "backend/api/internal/handler/rest"
```

Comment:
As i mentioned in another comment, plz help to sort this libs again by follow correct format